CauldronMods/Controller/Heroes/Gyrosaur/Cards/HiddenDetourCardController.cs

```diff
@@ -129,8 +129,30 @@ private IEnumerator AskToSwapCard(GameAction ga)
                 cardEnteringPlay = mc.CardToMove;
             }
 
-            var storedYesNo = new List<YesNoCardDecision>();
-            IEnumerator coroutine = GameController.MakeYesNoCardDecision(DecisionMaker, SelectionType.MoveCardToUnderCard, cardEnteringPlay, storedResults: storedYesNo, cardSource: GetCardSource());
+            Func<string> extraInfo;
+            List<Card> otherDetourCards = TurnTaker.GetCardsWhere((card) => card.Identifier == "HiddenDetour" && card.IsInPlayAndHasGameText && card != this.Card).ToList();
+            if (otherDetourCards.Count > 0)
+            {
+                string cardsUnder = "";
+                foreach (Card detour in otherDetourCards)
```

Comment:
Is it possible to have more than two Hidden Detours in play (or cards with the identifier "HiddenDetour")? There are only two in Gyro's deck, and anything that copies equipments probably doesn't have it's identifier changed. If there would only ever be two, I can simplify this block a bit. If there could be more than two, this block should catch it.